src/domUpdates.js

```diff
@@ -50,7 +50,10 @@ updateCurrentPlayerScore(player) {
 },
 
 appendLetter(guess) {
-  $(`#${guess}`).show();
+  // let puzzle = round.puzzle;
+  // console.log(puzzle)
+
+  $(`.${guess}`).show();
```

Comment:
I changed from a class to an id so that we can overwrite the disable feature for used letters. It wasn't working as we needed as a class.